packages/components/src/internal/util/Date.test.ts

```diff
@@ -355,7 +406,8 @@ describe('Date Utilities', () => {
         test('valid date with dateFormat', () => {
             expect(parseDate('01:02 2022-04-19', 'HH:mm yyyy-MM-dd').toString()).toContain('Apr 19 2022');
             expect(parseDate('19/04/2022', 'dd/MM/yyyy').toString()).toContain('Apr 19 2022');
-            expect(parseDate('4/11/2022', 'dd/MM/yyyy').toString()).toContain('Apr 11 2022');
+            // TODO: The following seems incorrect. I would expect November 4th (4th day of the 11th month). So does date-fns.
+            // expect(parseDate('4/11/2022', 'dd/MM/yyyy').toString()).toContain('Apr 11 2022');
```

Comment:
yeah that is odd. why was moment treating the "4" vs "04" so differently?

Reply:
No bueno.